velox/exec/SetAccumulator.h

```diff
@@ -86,6 +86,45 @@ struct SetAccumulator {
     }
   }
 
+  void addNullIndex(const char* buffer) {
+    VELOX_CHECK(!nullIndex.has_value());
+    vector_size_t serializedNullIndex;
+    memcpy(&serializedNullIndex, buffer, kVectorSizeT);
+    if (serializedNullIndex != kNoNullIndex) {
+      nullIndex = serializedNullIndex;
+    }
+  }
+
+  void addSerialized(
+      const FlatVector<StringView>& vector,
+      vector_size_t i,
+      HashStringAllocator* /*allocator*/) {
+    VELOX_CHECK(!vector.isNullAt(i));
+
+    // The serialized value is the nullOffset (kNoNullIndex if no null is
```

Comment:
It would be nice to document the serialization format in one place and avoid repeating it in comments.